compiler/src/dotty/tools/repl/ParseResult.scala

```diff
@@ -110,7 +111,7 @@ case object Help extends Command {
       |:type <expression>       evaluate the type of the given expression
       |:doc <expression>        print the documentation for the given expression
       |:imports                 show import history
-      |:reset                   reset the repl to its initial state, forgetting all session entries
+      |:reset [options]         reset the repl to its initial state, forgetting all session entries
       |:settings <options>      update compiler options, if possible
```

Comment:
super nitpicky, but: I think you meant angle brackets here, like the other entries?

Reply:
it's a type parameter, not an index.

Reply:
I copied Scala 2.  And I think I kind of grok the logic there to be "<options>" for settings, because it's expected to get options, even though it actually accepts none as a way to print options...  While for :reset (and :replay in the scala 2 repl) passing options is more of an extra, so more... optional (🥁 !)

Or we can deviate to fix it to some standard.  I just looked at http://docopt.org/ before realising I wasn't following that here, I was following what Scala 2 had.

Reply:
Personally I think it should follow docopt, but it's not a blocker.

Reply:
Is docopt like Coptic?